src/OWML.ModHelper.Menus/ModInputCombinationElementMenu.cs

```diff
@@ -102,7 +100,7 @@ public void Initialize(PopupInputMenu menu)
 
 			if (layout == null)
 			{
-				ModConsole.OwmlConsole.WriteLine("Error - Failed to create combination visualizer in combination editor.", MessageType.Error);
+				Console.WriteLine("Error - Failed to create combination visualizer in combination editor.", MessageType.Error);
```

Comment:
you've got unity code using system.console? or is this a variable named "Console"

Reply:
it's just named Console. VS tells me it's correct, it's not the static Console built-in to C#